docs/_static/config_examples/example-manifest.yml

```diff
@@ -44,3 +44,25 @@ applications:
                         uri: http://api.system.pcf.local
                         port: 80
                         auth_disabled: false
+
+                      # Required to run as a Service Broker
+                      status:
+                        user: user
+                        pass: pass
+
+      # Only required if going to push controller as a Service Broker
+      SERVICE_BROKER_CONFIG: |
+                             plans:
+                              - description: plan for policy A,
+                                name: planA,
+                                virtualServer:
+                                  - policies:
+                                    - policyA
+                                  - profiles:
+                                    - profileA
+                                  - sslProfiles:
+                                    - sslProfileA
+                                pool:
+                                  balance: ratio-member
+                                  healthMonitors:
+                                    - monitorA
```

Comment:
Add the following to the healthMonitors example:

```
- name: hm-test
  type: http
  interval: 12
  timeout: 5
  send: hello
```

Reply:
sounds good I'll add